packages/turf-boolean-point-on-line/index.d.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference types="geojson" />
+
+type Feature = GeoJSON.Feature<any> | GeoJSON.GeometryObject;
+
+/**
+ * http://turfjs.org/docs/#booleanPointOnLine
```

Comment:
All lowercase for the URL.

Example http://turfjs.org/docs/#centerOfMass (doesn't work)